src/csharp/Microsoft.Spark/Microsoft.Spark.csproj

```diff
@@ -15,4 +19,13 @@
     <PackageReference Include="System.Memory" Version="4.5.2" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta2-18618-05" PrivateAssets="All"/>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="..\..\scala\microsoft-spark-*\target\microsoft-spark-*.jar" Link="jars\%(Filename)%(Extension)"
+      Pack="true" PackagePath="jars\%(Filename)%(Extension)" />
```

Comment:
(nit) A more readable multi-line format here would be:

```xml
<Content Include="..\..\scala\microsoft-spark-*\target\microsoft-spark-*.jar"
                Link="jars\%(Filename)%(Extension)"
                Pack="true"
                PackagePath="jars\%(Filename)%(Extension)" />
```

It's easier to see each attribute. You don't have to parse the string, or miss something way off to the right.

Reply:
done